src/actions/scene.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Get scene name
+ * @param  {Eelement} scene Scene element
```

Comment:
Generally have done docstrings like:

```@param {Element} scene - Scene element.``` with only one space, a hyphen to separate argument name and desc, and end with period